typescript/simple-mercari-web/src/App.css

```diff
@@ -2,10 +2,22 @@ body{
   background-color: #222427;
```

Comment:
Maybe we can add all color values use of a color to make their usage more clear? 😊

--main-color: #222427;
  --title-color: #51cbf9;

Reply:
Fixed!